docker-compose.yml

```diff
@@ -48,6 +48,7 @@ services:
       - k6-net
     volumes:
       - ./config/influxdb/provisioning/users.sh:/docker-entrypoint-initdb.d/users.sh
+      - ./config/influxdb/influxdb.conf:/etc/influxdb/influxdb.conf
```

Comment:
is this required, we can use a env instead:

INFLUXDB_DATA_MAX_SERIES_PER_DATABASE=0

Reply:
I didn't find this on my first search. Now I changed the PR to this.